spec/features/user_deletes_outcome_coverage_from_a_subject_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+feature "User deletes an outcome coverage for a subject" do
+  scenario "coverage has 2 or more outcome coverages" do
+    course = create(:course)
+    outcome_A, outcome_B = create_pair(:outcome)
+    coverage = create(:coverage, course: course, outcomes: [outcome_A, outcome_B])
+    user = user_with_assessments_access_to(course.department)
+
+    visit manage_assessments_course_path(course.id, as: user)
+    first(".class-card-outcomes-wrapper").click_on("Delete")
+
+    expect(page).to have_outcome_number_of(1)
+  end
+
+  scenario "coverage has 1 outcome coverage with associated assignments" do
+    course = create(:course)
+    outcome = create(:outcome)
```

Comment:
Should this be associated with the course from the line above?

I think we could simplify a lot of this setup with some fancier factories, but that's out of scope for this PR.